opencti-platform/opencti-graphql/src/database/generate-message.js

```diff
@@ -56,12 +58,26 @@ export const generateDeleteMessage = (instance) => {
   return generateCreateDeleteMessage(EVENT_TYPE_DELETE, instance);
 };
 
-export const generateUpdateMessage = (entityType, inputs) => {
+export const generateUpdateMessage = async (context, entityType, inputs) => {
   const inputsByOperations = R.groupBy((m) => m.operation ?? UPDATE_OPERATION_REPLACE, inputs);
   const patchElements = Object.entries(inputsByOperations);
   if (patchElements.length === 0) {
     throw UnsupportedError('[OPENCTI] Error generating update message with empty inputs');
   }
+
+  const authorizedMembersIds = patchElements.slice(0, 3).flatMap(([,operations]) => {
+    return operations.slice(0, 3).flatMap(({ key, value }) => {
+      return key === 'authorized_members' ? (value ?? []).map(({ id }) => id) : [];
+    });
+  });
+  let members = [];
+  if (authorizedMembersIds.length > 0) {
+    members = await internalFindByIds(context, SYSTEM_USER, authorizedMembersIds, {
+      baseData: true,
+      baseFields: ['internal_id', 'name']
```

Comment:
nice thing to return only the fields we need 👍

I guess patchElements can have mutliple operations with authorized_members ?

Reply:
I'm not sure to understand the questions, you can change/add multiple authorized members at once yes

Reply:
ah yes ok, that is why I was wondering if we really need to log the authorized members name in history.